### PR TITLE
docs(Runbook): Update review dates on runbook

### PR DESCRIPTION
**Pull request by jackstockley89**
---
name: Cloud Platform Pull Request
about: Create new pull request for this repository

---

## Ticket

Tickets can be also referenced using "Connects to #NN" and "Closes #NN" syntax.
Please see Reference section for more information.

## Reference

[How to write good pull requests](https://www.annashipman.co.uk/jfdi/good-pull-requests.html)

[Github - closing issues with keywords](https://help.github.com/articles/closing-issues-using-keywords/)

### CODE REVIEW

File: runbooks/source/remove-data-from-elasticsearch.html.md.erb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Remove data from Elasticsearch
 weight: 190
-last_reviewed_on: 2023-03-16
+last_reviewed_on: 2023-07-25
 review_in: 3 months
```
Comment (jaskaransarkaria) [inline runbooks/source/remove-data-from-elasticsearch.html.md.erb:5]: Is it worth bumping this to 6 months too?

File: runbooks/source/cloud-platform-alliance.html.md.erb [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Cloud Platform Team Alliance
 weight: 5
-last_reviewed_on: 2023-03-31
+last_reviewed_on: 2023-07-26
 review_in: 3 months
```
Comment (jaskaransarkaria) [inline runbooks/source/cloud-platform-alliance.html.md.erb:5]: Is it worth bumping this to 6 months?

**Review by jaskaransarkaria — APPROVED** [review]

File: runbooks/source/remove-data-from-elasticsearch.html.md.erb (hunk 1 above)
Comment (jackstockley89) [reply runbooks/source/remove-data-from-elasticsearch.html.md.erb:5]: will bump this to 6 months

File: runbooks/source/cloud-platform-alliance.html.md.erb (hunk 2 above)
Comment (jackstockley89) [reply runbooks/source/cloud-platform-alliance.html.md.erb:5]: I left at 3 months as in the alliance it say we will review quarterly 

`We will review our alliance when the team changes or every quarter, whichever is sooner`

**Review by jaskaransarkaria — APPROVED** [review]

**Pull request merged**